shared/src/vol_interface.c

```diff
@@ -374,10 +360,10 @@ static void _image_flip_vertical( uint8_t* bytes_ptr, int width, int height, int
 /** Read the next frame of the video
  @returns   Pointer to the video frame pixel data
  */
-DllExport uint8_t * native_vol_read_next_frame(void)
+DllExport uint8_t * native_vol_read_next_video_frame( bool flip_vertical )
 {
     vol_av_read_next_frame( &video_file_ptr );
```

Comment:
i also made vflip optional so it doesnt do it on skipped frames any more - should help performance a bit.
we could add this as a param to vol_av_read_frame( vflip), so we dont have to have that code in our unity interface. (made a ticket)